### PR TITLE
refactor(advanced-tabs): refactored component to work with new tokens

### PR DESCRIPTION
**Pull request by daragh-king-genesys**
refactored component to work with new tokens

COMUI-1682

### CODE REVIEW

**Comment by daragh-king-genesys** [comment]
Any bugs that are not specifically code related can be reported with a comment in the JIRA issue https://inindca.atlassian.net/browse/COMUI-1682

File: packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced/gux-tab-advanced.tsx [hunk 1]
```diff
@@ -169,7 +198,9 @@ export class GuxTabAdvanced {
   }
 
   private toggleOptions(): void {
+    console.log('1, toggleOptions', this.popoverHidden);
```
Comment (katie-bobbe-genesys) [inline packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced/gux-tab-advanced.tsx:201]: remove console.logs before merging

File: packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced-list/gux-tab-advanced-list.tsx
Comment (katie-bobbe-genesys) [inline packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced-list/gux-tab-advanced-list.tsx:1]: Currently keyboard sort is broken, line 330 needs to be 
`this.sortTarget = (
              event.target as Element
            ).parentNode.parentNode.parentNode;`

Comment (daragh-king-genesys) [reply packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced-list/gux-tab-advanced-list.tsx:1]: I used
```
this.sortTarget = (event.target as Element).closest('gux-tab-advanced');
```

File: packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/example.html [hunk 2]
```diff
@@ -2,7 +2,7 @@ <h1>gux-tabs-advanced</h1>
 <h2>Example 1 - no add tab button</h2>
 <gux-tabs-advanced>
   <gux-tab-advanced-list slot="tab-list" show-new-tab-button="false">
-    <gux-tab-advanced tab-id="1-1">
+    <gux-tab-advanced tab-id="1-1" gux-disabled>
```
Comment (gavin-everett-genesys) [inline packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/example.html:5]: Is this change needed ?

File: packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced-list/gux-tab-advanced-list.scss [hunk 3]
```diff
@@ -2,138 +2,42 @@
...
-gux-tab-advanced-list {
-  .gux-tab-container {
+.gux-tab-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+  justify-content: flex-start;
+  width: 100%;
+  margin-bottom: 16px; //TODO
```
Comment (gavin-everett-genesys) [inline packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced-list/gux-tab-advanced-list.scss:12]: Is this `margin-bottom` is still needed ?

File: packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced-list/gux-tab-advanced-list.scss [hunk 4]
```diff
@@ -2,138 +2,42 @@
...
+    justify-content: flex-start;
     width: 100%;
-    height: 50px;
-    margin-bottom: 16px;
-    overflow: auto hidden;
-    background-color: #f6f7f9;
-    border-bottom: 1px solid #d7dce5;
-    border-radius: 4px 4px 0 0;
+    overflow-x: auto;
+    border-bottom: 1px solid var(--gse-ui-advancedTabs-divider-dividerColor); //TODO
```
Comment (gavin-everett-genesys) [inline packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced-list/gux-tab-advanced-list.scss:23]: I think this is the only one thats needs a token issue.

File: packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced-list/gux-tab-advanced-list.scss (hunk 3 above)
Comment (daragh-king-genesys) [reply packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced-list/gux-tab-advanced-list.scss:12]: Yes I have added it to the token issue https://inindca.atlassian.net/browse/COMUI-2444

File: packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced-list/gux-tab-advanced-list.scss (hunk 4 above)
Comment (daragh-king-genesys) [reply packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced-list/gux-tab-advanced-list.scss:23]: https://inindca.atlassian.net/browse/COMUI-2444

**Review by gavin-everett-genesys — APPROVED** [review]
